common/enum.utils.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * name : enum.utils.js
```

Comment:
@ankitpws This is not really enum, why are we calling it so? can be just constant like other services.